lib/alerts/components/CreateAlert.js

```diff
@@ -28,7 +30,7 @@ export default class CreateAlert extends Component<Props> {
         bsSize='large'
         style={{marginTop: '-8px'}}
         disabled={createDisabled}
-        onClick={createAlert}
```

Comment:
Shouldn't there be a linting error now that the `createAlert` is no longer used in the `render` method?

Reply:
Yes, I think you're right. And I have no idea why that commit would have done anything... Perhaps when I merged dev into this branch something got all funky, but that doesn't really explain the contents of the commit or the skipped lint.